x-pack/metricbeat/module/aws/s3_request/_meta/fields.yml

```diff
@@ -0,0 +1,74 @@
+- name: s3_request
+  type: group
+  description: >
+    `s3_request` contains request metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS S3.
+  release: beta
+  fields:
+    - name: bucket.name
+      type: keyword
+      description: >
+        Name of a S3 bucket.
+    - name: all_requests
+      type: long
+      description: >
+        The total number of HTTP requests made to an Amazon S3 bucket, regardless of type.
+    - name: get_requests
+      type: long
+      description: >
+        The number of HTTP GET requests made for objects in an Amazon S3 bucket.
+    - name: put_requests
+      type: long
+      description: >
+        The number of HTTP PUT requests made for objects in an Amazon S3 bucket.
+    - name: delete_requests
+      type: long
+      description: >
+        The number of HTTP DELETE requests made for objects in an Amazon S3 bucket.
+    - name: head_requests
+      type: long
+      description: >
+        The number of HTTP HEAD requests made to an Amazon S3 bucket.
+    - name: post_requests
+      type: long
+      description: >
+        The number of HTTP POST requests made to an Amazon S3 bucket.
+    - name: select_requests
+      type: long
+      description: >
+        The number of Amazon S3 SELECT Object Content requests made for objects in an Amazon S3 bucket.
+    - name: select_scanned.bytes
+      type: scaled_float
+      description: >
+        The number of bytes of data scanned with Amazon S3 SELECT Object Content requests in an Amazon S3 bucket.
+    - name: select_returned.bytes
+      type: scaled_float
+      description: >
+        The number of bytes of data returned with Amazon S3 SELECT Object Content requests in an Amazon S3 bucket.
+    - name: list_requests
+      type: long
+      description: >
+        The number of HTTP requests that list the contents of a bucket.
+    - name: bytes_downloaded
```

Comment:
`aws.s3_request.downloaded.bytes` I think I read somewhere that the unit is always at the end of the field name